### PR TITLE
PBI10.2: ranking board

### PR DESCRIPTION
**Pull request by TravisMai**

### CODE REVIEW

File: frontend/src/applications/play/RankingBoard.component.tsx [hunk 1]
```diff
@@ -32,9 +34,9 @@ export default function RankingBoard() {
   });
   const { codeRoom } = useParams();
 
-  const rankingOrder = (data: ILeaderboard[]) => {
-    return data.sort((a, b) => b.score - a.score);
-  };
+  // const rankingOrder = (data: ILeaderboard[]) => {
+  //   return data.sort((a, b) => b.score - a.score);
+  // };
```
Comment (khoatruong19) [inline frontend/src/applications/play/RankingBoard.component.tsx:39]: Xóa comment

Comment (TravisMai) [reply frontend/src/applications/play/RankingBoard.component.tsx:39]: yes sirrrrrr

File: frontend/src/applications/play/UserFrame.component.tsx [hunk 2]
```diff
@@ -26,12 +29,11 @@ const UserFrame: React.FC<ProfileProps> = (Leaderboard, _max_player) => {
     triggerRef.current?.click();
   };
   const renderItem = (data: ProfileProps) => {
-    console.log()
```
Comment (khoatruong19) [inline frontend/src/applications/play/UserFrame.component.tsx:29]: Xóa console

Comment (TravisMai) [reply frontend/src/applications/play/UserFrame.component.tsx:29]: no sirrr

**Pull request merged**